app/data/models/regular-job.js

```diff
@@ -23,11 +23,17 @@ module.exports = (sequelize, DataTypes) => {
     result: {
       type: DataTypes.STRING(1000),
       allowNull: false
+    },
```

Comment:
I don;t think we need audit columns on this table. Only used internally for job processing. No users would edit this table